---
layout: paper
title: "Enhancing immuno-oncology investigations through multidimensional decoding of tumor microenvironment with IOBR 2.0​"
nickname: 
authors: "Dongqiang Zeng, Yiran Fang, Wenjun Qiu, Peng Luo, Shixiang Wang, Rongfang Shen, Wenchao Gu, Xiatong Huang, Qianqian Mao, Gaofeng Wang, Yonghong Lai, Guangda Rong, Xi Xu, Min Shi, Zuqiang Wu, Guangchuang Yu, Wangjun Liao​"
year: "2024" 
journal:
volume: 4
issue: 12
pages: 100910
is_published: true
image:  /lab/assets/images/papers/2024-IOBR2.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC11704618
preprint:
supplement:

# Links
doi: 10.1016/j.crmeth.2024.100910
pmid: 39626665

# Data and code
github: []
neurovault: []
openneuro: []
osf:
figshare: []
---
{% include JB/setup %}

# Abstract

The use of large transcriptome datasets has greatly improved our understanding of the tumor microenvironment (TME) and helped develop precise immunotherapies. The growing application of multi-omics, single-cell RNA sequencing (scRNA-seq), and spatial transcriptome sequencing has led to many new insights, yet these findings still require clinical validation in large cohorts. To advance multi-omics integration in TME research, we have upgraded the Immuno-Oncology Biological Research (IOBR) package to IOBR 2.0, restructuring and standardizing its analytical workflow. IOBR 2.0 offers six modules for TME analysis based on multi-omics data, including data preprocessing, TME estimation, TME infiltration pattern identification, cellular interaction analysis, genome and TME interaction, and feature visualization, as well as modeling. Additionally, IOBR 2.0 enables constructing gene signatures and reference matrices from scRNA-seq data for TME deconvolution. The user-friendly pipeline provides comprehensive insights into tumor-immune interactions, and a detailed GitBook(https://iobr.github.io/book/) offers a complete manual and analysis guide for each module.
